SU2_CFD/src/hybrid_RANS_LES_model.cpp

```diff
@@ -584,10 +599,12 @@ void CHybrid_Mediator::ComputeInvLengthTensor(CVariable* flow_vars,
       for (jDim = 0; jDim < nDim; jDim++) {
         for (kDim = 0; kDim < nDim; kDim++) {
           // rotation contribution
-          Pij[iDim][jDim] += 2.0*alpha*eddy_viscosity*Sd_avg[iDim][kDim]*Om[kDim][jDim];
+          Pij[iDim][jDim] += 2.0*alpha_fac*eddy_viscosity*Sd_avg[iDim][kDim]*Om[kDim][jDim];
         }
         // rho*k contribtuion
-        Pij[iDim][jDim] -= 2.0*alpha*rho*ktot*(Sd[iDim][jDim]+Om[iDim][jDim])/3.0;
+	// should this part be just alpha scaling?
```

Comment:
Yes, yes it should.

Reply:
I agree, which is why the code is that way.  But, just food for thought, Sigfried says he gets better results on the periodic hill when the whole stress is scaled by a*(2-a), not just the anisotropic part. Given the definition of alpha, scaling by a*(2-a) on the isotropic part is clearly an inconsistency.  So... the fact that that works better must mean it is covering for some other issue.